### ISSUE TITLE
onnxmltools.convert_keras leads to parameters missing from target model

### ISSUE THREAD

**Issue by luludak** [onnx/onnxmltools#716]
I converted `InceptionV3` from Keras to ONNX, using `onnxmltools.convert_keras(....)`. The original model is one obtained from the pretrained models included in Keras API.

Upon deeper inspection I noticed that some pad hyperparameters are missing to specific `Conv` nodes. For example, Pads is missing in `target.Conv_34` vs `model/conv2d_5/BiasAdd`, as well as from `target.Conv_36` node compared to `model/conv2d_6/BiasAdd`. In some cases, there are `Pad` nodes added preceding the related `Conv` nodes, but that is not always the case and even if they are present, the padding value is different across the original and the converter model. Any ideas why this happens?
